pending/calibre.xml

```diff
@@ -31,4 +31,21 @@
     <action search="walk.all" command="delete" path="$LOCALAPPDATA\calibre-cache"/>
     <action search="walk.all" command="delete" path="~/.cache/calibre"/>
   </option>
+  <option id="logs">
+    <label>Logs</label>
+    <description>Delete the logs</description>
+    <action command="delete" search="file" path="~/.config/calibre/server_access_log.txt"/>
+    <action command="delete" search="file" path="~/.config/calibre/server_error_log.txt"/>
+  </option>
+  <option id="vacuum">
+    <label>Vacuum</label>
+     <description>Clean database fragmentation to reduce space and improve speed without removing any data</description>
+     <!-- NOTE: "~/Calibre Library/" is the default suggested location, however when first setting up Calibre, the user is
+     given the option of choosing a different location - many users may not have their library in the default location -->
```

Comment:
@tiemay : If you want me to take a look, open a new issue and post that json file (using a link to a gist or pastebin, if necessary).